# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/vm/TypedArrayObject.cpp
# Commit: 988eccbeb2ae
# Full Hash: 988eccbeb2ae3d4880be57237d2e3c51a4bdcd74
# Author: Steve Fink <sfink@mozilla.com>
# Date: 2021-09-09 15:22:41
# Regressor Bug: 1720422
# File Overlap Count: 1
# Description:
#   Bug 1720422 - Convert the JS_Is(type)Array APIs r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D119860
# ==============================================================================

diff -r 2d444b132a3d -r 988eccbeb2ae js/src/vm/TypedArrayObject.cpp
--- a/js/src/vm/TypedArrayObject.cpp	Wed Sep 08 23:46:59 2021 +0000
+++ b/js/src/vm/TypedArrayObject.cpp	Wed Sep 08 23:46:59 2021 +0000
@@ -2674,10 +2674,6 @@
     return obj;                                                               \
   }                                                                           \
                                                                               \
-  JS_PUBLIC_API bool JS_Is##Name##Array(JSObject* obj) {                      \
-    return js::Unwrap##Name##Array(obj) != nullptr;                           \
-  }                                                                           \
-                                                                              \
   JS_PUBLIC_API ExternalType* JS_Get##Name##ArrayLengthAndData(               \
       JSObject* obj, size_t* length, bool* isSharedMemory,                    \
       const JS::AutoRequireNoGC& nogc) {                                      \
